v20.2/drop-schedules.md

```diff
@@ -0,0 +1,70 @@
+---
+title: DROP SCHEDULES
```

Comment:
Worth a callout that `drop schedule` does NOT cancel any in progress jobs kicked off by the schedule, and they should cancel those jobs first because after you drop a schedule you can no longer look up their IDs